build/charts/ackdistro-monitor/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "v1.0"
+description: Trident monitor dashboard
+name: trident-monitor
```

Comment:
It should be "ackdistro" instead of "trident".

Reply:
Please fix all the other occurrences of the word "trident" including the file name.